safe/definitions/hazard_classifications.py

```diff
@@ -220,9 +220,9 @@
     ]
 }
 
-flood_hazard_classes = {
-    'key': 'flood_hazard_classes',
-    'name': tr('Flood classes'),
+flood_wet_dry_hazard_classes = {
+    'key': 'flood_wet_dry_hazard_classes',
```

Comment:
Just a note that this does not break anything....I updated all references to it.